pypgatk/commands/get_subpos.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+import click
+
+from pypgatk.toolbox.general import read_yaml_from_file
+from pypgatk.subpos.get_subpos import GetSubPos
+from pypgatk.commands.utils import print_help
+
+
+log = logging.getLogger(__name__)
+
+
+@click.command('get_subpos', short_help='Get position in peptide indicate which amino acid is substituted')
+@click.option('-c', '--config_file', help='Configuration file for the get subpos')
+@click.option('-in', '--input_psm_table', help='Input variant peptide PSMs table')
+@click.option('-i', '--input_fasta', help='Protein sequence used')
+@click.option('-o', '--output_psm_table', help='Output variant peptide PSMs table')
+@click.pass_context
+def get_subpos(ctx, config_file, input_psm_table, input_fasta, output_psm_table):
```

Comment:
@DongdongdongW I think this is a great idea. But do you think both things, `validate_peptides.py` and `get_subpos.py` can be merged together into only `validate_peptides.py`

Reply:
@ypriverol Do you mean to merge the two processes into the same process or just merge the code and merge the two processes under the same command? 
If it is the latter, I think it is more appropriate to put it under two separate commands, because it is two processes and one is the first and the other is the last. If it is the former, I also can easily do it, but users can no longer submit peptide table containing sub_pos. Even if it contains sub_pos, it still takes time to recalculate.